pyvista/plotting/renderer.py

```diff
@@ -505,15 +578,55 @@ def add_axes(self, interactive=None, line_width=2,
         interacitve : bool
             Enable this orientation widget to be moved by the user.
 
-        line_width : int
+        line_width : int, optional
             The width of the marker lines
 
-        box : bool
+        box : bool, optional
             Show a box orientation marker. Use ``box_args`` to adjust.
             See :any:`pyvista.create_axes_orientation_box` for details.
 
-        opacity : int or float, optional
-            The opacity of the marker.
+        xcolor : str, optional
```

Comment:
x_color, y_color, z_color.  Should we put the default behavior?

Reply:
The default is seen in the docstring.  We've established that it's optional to provide the defaults in our docstrings.